apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeEditor.tsx

```diff
@@ -524,7 +524,7 @@ export const CodeEditor = ({ sourcesFolderId, setValue }: Props) => {
             isOpen={!!deletingFileId}
             heading={t('Confirm deleting')}
             description={
-              t('Are you sure that you want to delete {{name}}', {
+              t('Are you sure that you want permanently delete {{name}}?', {
```

Comment:
I think it's incorrect word order and missing `to` and suggest this variant:
```suggestion
              t('Are you sure that you want to delete "{{name}}" permanently?', {
```